### ISSUE TITLE
SSL certificate validation disabled

### ISSUE THREAD

**Issue by frankie567** [mailjet/mailjet-apiv3-python#7]
Hi,

I was running tests on my code today involving Mailjet API calls and was surprised to get this warning:
```
Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/security.html
  InsecureRequestWarning)
```

Indeed, as we can see [in the api_call function](https://github.com/mailjet/mailjet-apiv3-python/blob/master/mailjet_rest/client.py#L87-L88), the `verify` flag is set to `False`. Is there any reason for this ? Wouldn't it be better to enforce SSL verification or at least let the developer enable it when instantiating a `Client` object ?

I could propose a PR for this if you wish 😃 

Regards !

**Comment by Zhivko-Mailjet** [comment]
HI @frankie567,

Thanks for spotting and reporting the issue. We definetely share your opinion that enforcing SSL verification is the right way to go and will work in this direction by setting the `verify` flag to `True`.  Hopefully, the update will be ready for the next week. 

Thanks again!

**Comment by arnaudbreton** [comment]
Hey @frankie567, good spot! I've just tested and fixed it. We're going to release a new version in the coming days, once PR #8 is merged. 

Thanks for your contribution,
Best

**Comment by frankie567** [comment]
Nice, thanks ! As I see, PR #8 will switch to API 3.1 ? That's nice, I'll be able to test the new Send API then ;)

**Comment by arnaudbreton** [comment]
@frankie567 yes, you bet! Please fill-in this [form](https://mailjet.typeform.com/to/XWMNNa) and I'll share with you all the details you need to use it.

**Comment by frankie567** [comment]
Well, actually, I've already done it yesterday (I'm François Voron, the guy who doesn't stop disturbing you both on GitHub and on support 😛 )! Thanks !

**Comment by arnaudbreton** [comment]
Oops, sorry! Didn't connect the dots. No worries, we're here to help :). Looking forward to hear back what you think about it!
